src/App.js

```diff
@@ -1,18 +1,25 @@
 import { useEffect, useState } from 'react'
 import { Route, Switch, useHistory } from 'react-router-dom'
 import CropDetailsPage from './pages/CropDetailsPage'
+import FormPage from './pages/FormPage'
 import MyGardenPage from './pages/MyGardenPage'
 import ResultsPage from './pages/ResultsPage'
 import SearchPage from './pages/SearchPage'
+import TasksPage from './pages/TasksPage'
 import { loadFromLocal, saveToLocal } from './utils/localStorage'
 
 export default function App() {
   const history = useHistory()
+  const [taskList, setTaskList] = useState(loadFromLocal('taskList') ?? [])
 
   const [favoriteCrops, setFavoriteCrops] = useState(
     loadFromLocal('favoriteCrops') ?? []
   )
 
+  useEffect(() => {
+    saveToLocal('taskList', taskList)
+  }, [taskList])
+
   useEffect(() => {
     saveToLocal('favoriteCrops', favoriteCrops)
   }, [favoriteCrops])
```

Comment:
```suggestion
  const [taskList, setTaskList] = useState(loadFromLocal('taskList') ?? [])
  
    useEffect(() => {
    saveToLocal('taskList', taskList)
  }, [taskList])

  const [favoriteCrops, setFavoriteCrops] = useState(
    loadFromLocal('favoriteCrops') ?? []
  )
  
  useEffect(() => {
    saveToLocal('favoriteCrops', favoriteCrops)
  }, [favoriteCrops])
```

just swapped it for better readability.